src/Microsoft.Identity.Client/Cache/Keys/MsalAccountCacheKey.cs

```diff
@@ -34,14 +34,15 @@ namespace Microsoft.Identity.Client.Cache.Keys
     /// An object representing the key of the token cache Account dictionary. The 
     /// format of the key is not important for this library, as long as it is unique.
     /// </summary>
-    internal class MsalAccountCacheKey 
+    internal class MsalAccountCacheKey : IiOSKey
```

Comment:
curious question:  should we perhaps have the key generator class come from PlatformProxy so that we can generate the appropriate keys if they're different on a platform like iOS?  Could be weird if we want to support serialized keys and other such stuff, but I think having some type of factory for "give me the appropriate key generator for my platform and scenario" and returning an interface that can be used as appropriate might make some of this cleaner.

Reply:
Yes good point, if I have time I will do this.

Reply:
Discussed with Henrik as well, will address in a separate PR.